src/main/java/at/hannibal2/skyhanni/features/misc/GeorgeDisplay.kt

```diff
@@ -0,0 +1,86 @@
+package at.hannibal2.skyhanni.features.misc
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.InventoryOpenEvent
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.cleanName
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.NEUItems.getPriceOrNull
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.RenderUtils.renderRenderables
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.renderables.Renderable
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class GeorgeDisplay {
+
+    private val config get() = SkyHanniMod.feature.misc.pets.george
+
+    private val ROW_OFFSET: Int = 9 * 4
```

Comment:
> Is there a reason

given hypixel's way of moving things around in menus without mod developer consent it's easier to split the slot ID into individual math expressions. ie: `9 * 4` to skip the first four rows, and `6 - 1` because the spawn egg is on the sixth slot of that row (and minus one because of "starts at zero" index)

Reply:
alright im going to hardcode the item slot but if the slot ID changes, don't egg my house